src/SFA.DAS.Apprenticeships.Application/Services/ApprenticeshipService.cs

```diff
@@ -205,37 +184,37 @@ public async Task CreateStartDateChange(Guid apprenticeshipKey, string initiator
     public async Task RejectPendingStartDateChange(Guid apprenticeshipKey, string reason)
     {
 	    var response = await _apiClient.Patch<object>(new RejectPendingStartDateChangeRequest(apprenticeshipKey, new RejectPendingStartDateChangeData { Reason = reason }));
-	    if (!string.IsNullOrEmpty(response?.ErrorContent))
-	    {
-		    throw new ServiceException(response.ErrorContent);
-	    }
+        ValidateResponse(response, nameof(RejectPendingStartDateChange));
     }
 
     public async Task CancelPendingStartDateChange(Guid apprenticeshipKey)
     {
 		var response = await _apiClient.Delete<object>(new CancelPendingStartDateChangeRequest(apprenticeshipKey));
-		if (!string.IsNullOrEmpty(response.ErrorContent))
-		{
-			throw new ServiceException(response.ErrorContent);
-		}
-	}
+        ValidateResponse(response, nameof(CancelPendingStartDateChange));
+    }
 
     public async Task FreezePayments(Guid apprenticeshipKey, string? reason)
     {
         var response = await _apiClient.Post<object>(new FreezePaymentsRequest(apprenticeshipKey, new FreezePaymentsData{ Reason = reason}));
-        if (!response.IsSuccessStatusCode)
-        {
-            throw new ServiceException($"Failed to {nameof(FreezePayments)} Status:{response.StatusCode} Content: {response.ErrorContent}");
-        }
+        ValidateResponse(response, nameof(FreezePayments));
     }
 
     public async Task UnfreezePayments(Guid apprenticeshipKey)
     {
         var response = await _apiClient.Post<object>(new UnfreezePaymentsRequest(apprenticeshipKey));
+        ValidateResponse(response, nameof(UnfreezePayments));
+    }
+
+    private void ValidateResponse<T>(ApiResponse<T> response, string methodName)
+    {
+        if (response == null)
```

Comment:
GetApprenticeshipStartDate isn't calling this method, only the post/patch/delete methods are calling it. The get endpoints all have existing manual handling of responses. I could change the apiResponse object so that get requests can't use this method to prevent it being used for get methods in the future